tests/typeConverter.test.ts

```diff
@@ -932,3 +932,157 @@ export type UserOperation = 'Read' | 'Write';
 `;
     expect(resultString).toEqual(expectedString);
 });
+
+it('should return overrided enum schema', async () => {
+
+    // What will be fetched from Swagger Json
+    const example = {
+        components: {
+            schemas: {
+                ServiceOfferKind: {
+                    type: 'string',
+                    description: '',
+                    'x-enumNames': [
+                        'MasteringAndDistribution',
+                        'Video',
+                        'Samples',
+                        'Mastering',
+                        'Distribution',
+                        'Sessions',
+                    ],
+                    enum: ['MasteringAndDistribution', 'Video', 'Samples', 'Mastering', 'Distribution', 'Sessions'],
+                },
+            },
+        },
+    };
+
+    const resultString = parseSchemas({
+        json: example,
+        swaggerVersion: 3,
+        // Overrided value "ServiceOfferKind" enum
+        overrideSchemas: [
+            {
+                ServiceOfferKind: {
+                    type: 'string',
```

Comment:
it's not type `enum` ?

Reply:
@ardeois this is related to Swagger annotation. as you can see at line 968 https://github.com/LandrAudio/openapi-codegen-typescript/pull/18/files/78ee1727af4c49049fa27628ed55d8c675135213#diff-d21f8ad994231515879709a1173cbbc1967f8f5c1f71ff40e9ef448b0de4dd73R953 it has enum property.